cmd/telegraf/cmd_win_service.go

```diff
@@ -0,0 +1,202 @@
+						fmt.Fprintf(outputBuffer, "Successfully installed service %q\n", name)
+						return nil
+					},
+				},
+				{
+					Name:  "uninstall",
+					Usage: "remove the Telegraf Windows service",
+					Description: `
+The 'uninstall' command removes the Telegraf service with the given name. To
+remove a service use
+
+> telegraf service uninstall
+
+In case you specified a custom service-name during install use
+
+> telegraf --service-name telegraf-machine service uninstall
+`,
+					Action: func(cCtx *cli.Context) error {
+						name := cCtx.String("service-name")
+						fmt.Fprintf(outputBuffer, "Uninstalling service %s...\n", name)
+						if err := uninstallService(name); err != nil {
+							return err
+						}
+						fmt.Fprintf(outputBuffer, "Successfully uninstalled service %q\n", name)
+						return nil
+					},
+				},
+				{
+					Name:  "start",
+					Usage: "start the Telegraf Windows service",
+					Description: `
+The 'start' command triggers the start of the Windows service with the given
+name. To start the service either use the Windows service manager or run
+
+> telegraf service start
+
+In case you specified a custom service-name during install use
+
+> telegraf --service-name telegraf-machine service start
+`,
+					Action: func(cCtx *cli.Context) error {
+						name := cCtx.String("service-name")
+						fmt.Fprintf(outputBuffer, "Starting service %s...\n", name)
+						if err := startService(name); err != nil {
+							return err
+						}
+						fmt.Fprintf(outputBuffer, "Successfully started service %q\n", name)
+						return nil
+					},
+				},
+				{
+					Name:  "stop",
+					Usage: "stop the Telegraf Windows service",
+					Description: `
+The 'stop' command triggers the stop of the Windows service with the given
+name and will wait until the service is actually stopped. To stop the service
+either use the Windows service manager or run
+
+> telegraf service stop
+
+In case you specified a custom service-name during install use
+
+> telegraf --service-name telegraf-machine service stop
+`,
+					Action: func(cCtx *cli.Context) error {
+						name := cCtx.String("service-name")
+						fmt.Fprintf(outputBuffer, "Stopping service %s...\n", name)
+						if err := stopService(name); err != nil {
+							return err
+						}
+						fmt.Fprintf(outputBuffer, "Successfully stopped service %q\n", name)
+						return nil
+					},
+				},
+				{
+					Name:  "status",
+					Usage: "query the Telegraf Windows service status",
+					Description: `
+The 'status' command queries the current state of the Windows service with the
+given name. To query the service either check the Windows service manager or run
+
+> telegraf service status
+
+In case you specified a custom service-name during install use
+
+> telegraf --service-name telegraf-machine service status
+`,
+					Action: func(cCtx *cli.Context) error {
+						name := cCtx.String("service-name")
+						fmt.Fprintf(outputBuffer, "Querying status of service %s...\n", name)
```

Comment:
I think we can remove these as the user should know what they are doing if they run the subcommand `status`.